package.json

```
@@ -7,17 +7,17 @@
     "url": "https://github.com/ably/ably-js/issues",
     "email": "support@ably.com"
   },
-  "main": "./browser/static/ably-node.js",
-  "typings": "./ably.d.ts",
+  "main": "./build/ably-node.js",
+  "typings": "./build/ably.d.ts",
   "react-native": {
-    "./browser/static/ably-node.js": "./browser/static/ably-reactnative.js"
+    "./build/ably-node.js": "./build/ably-reactnative.js"
   },
   "browser": {
-    "./browser/static/ably-node.js": "./browser/static/ably-commonjs.js"
+    "./build/ably-node.js": "./build/ably-commonjs.js"
   },
   "files": [
+    "src/**",
```

The `files` field specifies which files should be included in the npm package so we need to keep the build output in here.
```suggestion
    "build/**",
```